src/jira/client/jira-client.ts

```diff
@@ -505,8 +517,14 @@ const updateRepositoryIssueKeys = (repositoryObj, mutatingFunc) => {
  * Runs the mutatingFunc on the issue keys field for each branch, commit or PR
  */
 const updateIssueKeysFor = (resources, func) => {
```

Comment:
You're the Typescript expert and this looks like the end of a long line of non - typed events, is there any value adding :any types to `resources` and `func`?

Reply:
yeah, agreed, this needs to be fixed, but the whole files is bad and there's a ticket to replace it/fix it already.  There's no reason to add any as it's already assumed.